example/Swift/PrebidDemo/PrebidDemo/BannerController.swift

```diff
@@ -32,17 +32,17 @@ class BannerController: UIViewController, GADBannerViewDelegate, MPAdViewDelegat
     let request = DFPRequest()
     
     var dfpBanner: DFPBannerView!
-    var bannerUnit: BannerAdUnit!
     
     var mopubBanner: MPAdView?
+    var dispatcher: Dispatcher?
     
     override func viewDidLoad() {
         super.viewDidLoad()
         
         adServerLabel.text = adServerName
         
-        bannerUnit = BannerAdUnit(configId: "6ace8c7d-88c0-4623-8117-75bc3f0a2e45", size: CGSize(width: 300, height: 250))
-        bannerUnit.setAutoRefreshMillis(time: 35000)
+        let bannerUnit = BannerAdUnit(configId: "6ace8c7d-88c0-4623-8117-75bc3f0a2e45", size: CGSize(width: 300, height: 250))
+        dispatcher = bannerUnit.setAutoRefreshMillis(time: 35000)
```

Comment:
This deviates from our intend to implement. Dispatcher is not a public class... this should be used internal to prebid mobile